ts/src/btcmarkets.ts

```diff
@@ -86,7 +86,7 @@ export default class btcmarkets extends Exchange {
                 'withdraw': true,
             },
             'urls': {
```

Comment:
there's a typo here `ttp` instead of `http`

Reply:
> there's a typo here `ttp` instead of `http`

thanks! changed to 'https' instead of 'ttps'